cdm/core/src/main/java/ucar/nc2/ft/point/writer/WriterCFStationCollection.java

```diff
@@ -63,14 +61,13 @@ public WriterCFStationCollection(String fileOut, List<Attribute> atts, List<Vari
         "Timeseries of station data in the indexed ragged array representation, H.2.5"));
   }
 
-  public void writeHeader(StationTimeSeriesFeatureCollection stations) throws IOException {
+  public void writeHeader(StationFeatureCollection stations) throws IOException {
```

Comment:
This is modifying the signature of a public class but StationTimeSeriesFeatureCollection extends StationFeatureCollection so it shouldn't cause any problems.